install_deps.cmake

```diff
@@ -63,21 +65,24 @@ if(NOT CMakeGet_FOUND)
     file(DOWNLOAD https://raw.githubusercontent.com/pfultz2/cmake-get/master/install.cmake ${FILENAME} STATUS RESULT_LIST)
     list(GET RESULT_LIST 0 RESULT)
     list(GET RESULT_LIST 1 RESULT_MESSAGE)
+
     if(NOT RESULT EQUAL 0)
         message(FATAL_ERROR "Download for install.cmake failed: ${RESULT_MESSAGE}")
     endif()
+
     execute_process(COMMAND ${CMAKE_COMMAND} -P ${FILENAME} ${PREFIX})
     file(REMOVE ${FILENAME})
     find_package(CMakeGet REQUIRED PATHS ${PREFIX})
 endif()
 
-# Set compiler to hcc if not set
+# Set compiler to clang++ if not set
```

Comment:
with this change, it seems like we can update the README to remove `CXX=/opt/rocm/llvm/bin/clang++` from the cmake command?

Reply:
I don't think so. `install_deps.cmake` is another way to install dependencies. 

We are not using or don't have documented way of using `install_deps.cmake` to install dependencies. 

If users haven't installed dependencies using this `install_deps.cmake` then, they won't have `CXX` environment variable.  

For example, MIOpen uses this : https://github.com/ROCmSoftwarePlatform/MIOpen#installing-the-dependencies